packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -85,6 +84,11 @@ import {
 import { nls } from '../messages';
 import os = require('os');
 
+// Below import has to be required for bundling
+/* tslint:disable */
+const AsyncLock = require('async-lock');
+/* tslint:enable */
```

Comment:
There are few different files that required this kind of update.  Basically the async-lock dep doesn't bundle well b/c it doesn't have a default export.